packages/account-v2/src/modules/AddressFields/AddressFields.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useAuthorize, useSettings, useStatesList } from '@deriv/api';
+import FormDropDownField from '../../components/FormFields/FormDropDownField';
+import FormInputField from '../../components/FormFields/FormInputField';
+import { addressDetailValidations } from './validations';
+
+export const AddressFields = () => {
+    const { data: activeAccount } = useAuthorize();
+    const { data: settings } = useSettings();
+
+    const { landing_company_name: landingCompanyName, upgradeable_landing_companies: upgradableLandingCompanies } =
+        activeAccount;
+
+    const isSvg = landingCompanyName === 'svg' || !!upgradableLandingCompanies?.includes('svg');
+    const { data: statesList, isFetched: statesListFetched } = useStatesList(settings.country_code || '', {
+        enabled: !!settings.country_code,
+    });
+
+    const {
+        addressCity: addressCitySchema,
+        addressLine1: addressLine1Schema,
+        addressLine2: addressLine2Schema,
+        addressPostcode: addressPostcodeSchema,
+        addressState: addressStateSchema,
+    } = addressDetailValidations(settings.country_code || '', isSvg);
```

Comment:
settings?.country_code

Reply:
settings cannot be undefined because we are creating a modified object in the hook, country_code can be undefined that's why need null coalescing here.